### PR TITLE
update docs for identity protection v1.0

### PR DESCRIPTION
**Pull request by cloudhandler**

### CODE REVIEW

**Comment by Lauragra** [comment]
@cloudhandler , please take a look at the API Doctor errors in the validation report. We can't merge this PR until all the validation errors are fixed.

**Comment by cloudhandler** [comment]
> @cloudhandler , please take a look at the API Doctor errors in the validation report. We can't merge this PR until all the validation errors are fixed.

I'm not sure what to do for these 3 errors, as the types are defined within the doc? Is there somewhere else they need to be defined as well?
image

**Comment by Lauragra** [comment]
> > @cloudhandler , please take a look at the API Doctor errors in the validation report. We can't merge this PR until all the validation errors are fixed.
> 
> I'm not sure what to do for these 3 errors, as the types are defined within the doc? Is there somewhere else they need to be defined as well?
> image

Please ping the Docs - API Doctor channel for help with the errors.

**Comment by cloudhandler** [comment]
> > > @cloudhandler , please take a look at the API Doctor errors in the validation report. We can't merge this PR until all the validation errors are fixed.
> > 
> > 
> > I'm not sure what to do for these 3 errors, as the types are defined within the doc? Is there somewhere else they need to be defined as well?
> > image
> 
> Please ping the Docs - API Doctor channel for help with the errors.

Thanks, posted in the API Doctor channel yesterday but still haven't heard back :(. Hopefully get a response today as this is currently rolling out

File: api-reference/v1.0/api/riskyuser-list-history.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskyuserhistoryitem)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskyuser-list-history.md:74]: Add new line between headers and body

File: api-reference/v1.0/api/riskyuser-list.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskyuser)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskyuser-list.md:73]: Add new line between headers and body

File: api-reference/v1.0/api/riskdetection-list.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskdetection)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskdetection-list.md:76]: Add new line between headers and body

Comment (cloudhandler) [reply api-reference/v1.0/api/riskdetection-list.md:77]: updated

File: api-reference/v1.0/api/riskyuser-list-history.md (hunk 1 above)
Comment (cloudhandler) [reply api-reference/v1.0/api/riskyuser-list-history.md:75]: updated

File: api-reference/v1.0/api/riskyuser-list.md (hunk 2 above)
Comment (cloudhandler) [reply api-reference/v1.0/api/riskyuser-list.md:74]: updated

File: api-reference/v1.0/api/riskdetection-list.md (hunk 3 above)
Comment (cloudhandler) [reply api-reference/v1.0/api/riskdetection-list.md:77]: @millicentachieng - the API doctor errors are saying there's an issue with the get topics?
image

Comment (cloudhandler) [reply api-reference/v1.0/api/riskdetection-list.md:77]: It looks like that fixed the errors on the space, but now there's a new error " Unable to locate a definition for resource type"
image

File: api-reference/v1.0/api/riskyuser-list.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+GET https://graph.microsoft.com/v1.0/identityProtection/riskyUsers
+```
+
+
+### Response
+**Note:** The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskyuser)"
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskyuser-list.md:67]: Case of type should be consistent. Should be `riskyUser`.

File: api-reference/v1.0/api/riskyuser-list-history.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+GET https://graph.microsoft.com/v1.0/identityProtection/riskyUsers/{riskyUserId}/history
+```
+
+
+### Response
+**Note:** The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskyuserhistoryitem)"
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskyuser-list-history.md:68]: Should be `riskyUserHistoryItem` at the end

File: api-reference/v1.0/api/riskdetection-list.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+GET https://graph.microsoft.com/v1.0/identityProtection/riskDetections
+```
+
+
+### Response
+**Note:** The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.riskdetection)"
```
Comment (millicentachieng) [inline api-reference/v1.0/api/riskdetection-list.md]: Should match the case of the type, `riskDetection`

Comment (cloudhandler) [reply api-reference/v1.0/api/riskdetection-list.md]: fixed

File: api-reference/v1.0/api/riskyuser-list-history.md (hunk 5 above)
Comment (cloudhandler) [reply api-reference/v1.0/api/riskyuser-list-history.md]: fixed

File: api-reference/v1.0/api/riskyuser-list.md (hunk 4 above)
Comment (cloudhandler) [reply api-reference/v1.0/api/riskyuser-list.md]: fixed

**Comment by cloudhandler** [comment]
@Lauragra - API doctor errors have been fixed, we're ready to merge :)

File: api-reference/v1.0/resources/riskyuser.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+localization_priority: Normal
+ms.prod: "microsoft-identity-platform"
+doc_type: resourcePageType
+---
+
+# riskyUser resource type
+
+Namespace: microsoft.graph
+
+**TODO: Add Description**
```
Comment (Lauragra) [inline api-reference/v1.0/resources/riskyuser.md:14]: This resource is missing a description.

File: api-reference/v1.0/api/riskyuser-list.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+Namespace: microsoft.graph
+
+Get a list of the [riskyUser](../resources/riskyuser.md) objects and their properties.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions_reference).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
```
Comment (Lauragra) [inline api-reference/v1.0/api/riskyuser-list.md:20]: Please address to the TODOs.

File: api-reference/v1.0/resources/riskyuser.md (hunk 7 above)
Comment (cloudhandler) [reply api-reference/v1.0/resources/riskyuser.md]: updated, thanks!

**Comment by cloudhandler** [comment]
@Lauragra - with the changes, are we good to merge? The blog goes live tomorrow morning. Thanks!

**Pull request merged**